src/controllers/expenses.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+const usersService = require('../services/users');
+
+const getAllFiltered = (req, res) => {
+  const expenses = expensesService.filterExpenses(req.query);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expensesService.getExpenseById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const add = (req, res) => {
+  const {
+    userId,
+    title,
+  } = req.body;
+
+  const checkOnUser = usersService.getUserById(userId);
```

Comment:
Please check for userId and title separately, and send a more informative message with the 400 status regarding which parameter is missing.